frontend/src/views/Game/Ingame.svelte

```diff
@@ -9,10 +9,76 @@
   export let socket: SocketClient
   export let gameData: GameSocketResponse
   export let currentUser: UserResponse
+  enum GameState {
+    PLAYING = 'playing',
+    FLIPPING = 'flipping',
+    VOTING = 'voting',
+    DISPLAY_WINNER = 'winner',
+  }
 
-  const playCard = (card: CardResponse) => {
-    if(!socket.currentUser.cardWizz) {
-      socket.playCard(card)
+  const getWinningPlayer = (game: GameSocketResponse): UserResponse => {
+    const winningPlayer = getPlayers(game).find(user => user.cards.some(c => c.state === CardState.WINNER))
+    if(!winningPlayer) {
+      throw new Error('Could not find winning player')
+    }
+    return winningPlayer
+  }
+
+  const getPlayers = (game: GameSocketResponse): UserResponse[] => {
+    return game.users.filter(user => !user.cardWizz)
+  }
+
+  const getPlayedCards = (game: GameSocketResponse): CardResponse[] => {
+    let allPlayedCards: CardResponse[] = []
+    for(const player of getPlayers(game)) {
+      allPlayedCards = [...allPlayedCards, ...player.cards.filter(card => card.state !== CardState.HAND)]
```

Comment:
`!=== HAND` watch out for this, since we have a `USED` state which might cause bugs here

Reply:
add a todo on this